examples/tutorials/plot_timeseries_example.py

```diff
@@ -188,4 +190,18 @@
 # the column using the unit keyword:
 qua_new = u.Quantity(qua.value * 0.00001, ts_eve.units[colname])
 unit = u.W/(u.km**2)
-ts_eve = ts_eve.add_column(colname, qua_new, unit=unit, overwrite=True)
\ No newline at end of file
+ts_eve = ts_eve.add_column(colname, qua_new, unit=unit, overwrite=True)#
+
+##############################################################################
+# MetaData
+# Get
+date_obs = combined_goes_ts.meta.get('date-obs', [ ])
+date_obs = combined_goes_ts.meta.get('date-obs', [ ], datetime=parse_time('2012-06-02T21:08:12'))

```

Comment:
It's just when making examples I tend to start them off being explicit. It does check for a string and use parse_time when applicable.